Combinatorial_map/doc/Combinatorial_map/Concepts/CombinatorialMap.h

```diff
@@ -15,72 +15,72 @@ class CombinatorialMap {
   0 \f$ \leq\f$ <I>j</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink
-  and `*dh`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
+  and `d`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
 */
-Dart_handle beta(Dart_handle dh, int i, int j);
+Dart_descriptor beta(Dart_descriptor d, int i, int j);
 
 /*!
-Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`*dh`)).
+Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`d`)).
 Overloads of this member function are defined that take from one to nine integer as arguments.
 \pre 0 \f$ \leq\f$ <I>i</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink,
      0 \f$ \leq\f$ <I>j</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink
-     and `*dh`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
+     and `d`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
 
 */
-Dart_const_handle beta(Dart_const_handle dh, int i, int j) const;
+Dart_const_descriptor beta(Dart_const_descriptor d, int i, int j) const;
 
 /*!
-Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`*dh`)).
+Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`d`)).
 Overloads of this member function are defined that take from one to nine integer as template arguments.
 For each function, betas are applied in the same order as their indices are given as template arguments.
 
-For example `beta<1>(dh)`=\f$ \beta_1\f$(`*dh`),
-and `beta<1,2,3,0>(dh)`=\f$ \beta_0\f$(\f$ \beta_3\f$(\f$ \beta_2\f$(\f$ \beta_1\f$(`*dh`)))).
+For example `beta<1>(d)`=\f$ \beta_1\f$(`d`),
+and `beta<1,2,3,0>(d)`=\f$ \beta_0\f$(\f$ \beta_3\f$(\f$ \beta_2\f$(\f$ \beta_1\f$(`d`)))).
 \pre 0 \f$ \leq\f$ <I>i</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink,
      0 \f$ \leq\f$ <I>j</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink
-  and `*dh`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
+  and `d`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
 */
 template<int i, int j>
-Dart_handle beta(Dart_handle dh);
+Dart_descriptor beta(Dart_descriptor d);
 
 /*!
-Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`*dh`)).
+Returns \f$ \beta_j\f$(\f$ \beta_i\f$(`d`)).
 Overloads of this member function are defined that take from one to nine integer as template arguments.
 \pre 0 \f$ \leq\f$ <I>i</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink,
      0 \f$ \leq\f$ <I>j</I> \f$ \leq\f$ \link GenericMap::dimension `dimension`\endlink
-     and `*dh`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
+     and `d`\f$ \in\f$ \link GenericMap::darts `darts()`\endlink.
 
 */
 template<int i, int j>
-Dart_const_handle beta(Dart_const_handle dh) const;
+Dart_const_descriptor beta(Dart_const_descriptor d) const;
 
 /*!
-Returns a handle to a dart belonging to the same edge than dart `*dh`, and not to the same vertex. `nullptr` if such a dart does not exist.
+Returns a descriptor to a dart belonging to the same edge than dart `d`, and not to the same vertex. `null_descriptor` if such a dart does not exist.
```

Comment:
Should it be `null_dart_decriptor` here? If yes, please check all of them.

Reply:
No this is null_descriptor (equivalent to nullptr for indices).

null_dart_descriptor is a descriptor to the special dart used to represent borders.